src/features/pages/partners/deriv-prime/commercial-plan/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { commercial_table } from './styles.module.scss'
+import Container from 'features/components/atoms/container'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Table from 'features/components/atoms/table'
+import dclsx from 'features/utils/dclsx'
+
+const tableData = [
```

Comment:
Can we change the data name to `snake_case` eg: table_data

Reply:
Sure..